yt_dlp/extractor/bilibili.py

```diff
@@ -51,16 +51,21 @@ def extract_formats(self, play_info):
         flac_audio = traverse_obj(play_info, ('dash', 'flac', 'audio'))
         if flac_audio:
             audios.append(flac_audio)
+        # dolby audio (if exists)
+        audios.extend(traverse_obj(play_info, ('dash', 'dolby', 'audio', ...)))
```

Comment:
this can be included in the original `audios` traversal instead
```py
        audios = traverse_obj(play_info, ('dash', (None, 'dolby'), 'audio', ..., {dict}))
```